app/resolve/daemon/__init__.py

```diff
@@ -5,12 +5,26 @@
 from core.parser import parser
 from core.rquest_dto.result import RquestResult
 from core.task_api_client import TaskApiClient
+from core.results_modifiers import result_modifiers
+import json
 
 
 def main() -> None:
     client = TaskApiClient()
     logger = logging.getLogger(settings.LOGGER_NAME)
-    result = execute_query(parser)
+    modifiers_list = result_modifiers(
```

Comment:
will this exclude them from the list if the env variable is not set? it looks like it sets the value to 0, but they should be absent/`None`?

Reply:
This is true - if you set it to 0 it tries to divide by 0

Reply:
done

Reply:
yes the modifiers builder will not add the filter to the list if the corresponding value is not set/ 0, because Python will treat it as 'false" and ignore the append command

Reply:
in here
![image](https://github.com/user-attachments/assets/13fb0c3d-66df-44b0-9502-3760ac13d891)